### DISCUSSION TITLE
Python 3.11 Support

### DISCUSSION THREAD

**Discussion by theCapypara** [ymyzk/tox-gh-actions#151] [Q&A] [answered]
**Describe the bug**
This does currently not support running Python 3.11 environments.

**To Reproduce**
Add an `3.11: py311` environment to the `[gh-actions]` sections and run the tests from within a Github action with Python 3.11.

**Expected behavior**
I expect the tests to run.

**Additional context**
Tox immediately exists, printing `Warning: WARNING: tox-gh-actions couldn't find environments matching the provided factors from envlist. Please use `tox -vv` to get more detailed logs.`.

**Comment by ymyzk** [comment]
Hello @theCapypara, thanks for filing a ticket. Even though the latest version doesn't explicitly say Python 3.11 is supported, it should run on Python 3.11 without major issues. I couldn't actually reproduce your issue in my environment. Could you share your `tox.ini` and logs from `tox -vv` if possible?

**Comment by theCapypara** [comment, marked as answer]
Hm, I set up a small test repo and am unable to reproduce this myself. But thanks for letting me know it's supposed to work, I'll investigate why my particular case doesn't.
